src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+extends _blog.pug
+
+block append subtitle
+  | Year one recap
+
+block content
+  h2 Year one recap
+  +news_header('22 November 2023', 'report_18', '')
+
+  p One year ago Input Labs went live, it is time to review our achievements and our failures.
```

Comment:
Full stop after live.
Either end after "achievements" or something else for "failures", e.g. "what could have gone better".